src/main/java/ladder/console/ConsoleOutput.java

```diff
@@ -0,0 +1,48 @@
+package ladder.console;
+
+import ladder.dto.BlockDto;
+import ladder.dto.LadderResult;
+
+public class ConsoleOutput {
+    public static final String POLL = "-----";
+    public static final String FRAME = "|";
+    public static final String PADDING = "     ";
+    public static final char NEW_LINE = '\n';
+
+    public static void drawResult(LadderResult result) {
+        showResult();
+
+        StringBuilder stringBuilder = new StringBuilder();
+
+        appendUsers(result, stringBuilder);
+        appendBlocks(result, stringBuilder);
```

Comment:
(!)스타일있을수도있습니다 😄 
객체에 가해지는 행동을 함수내부로 숨기기보단, 
겉으로 드러내는것을 선호해서, 다음과 같이도 고려해보시면 어떨까요 👍 !

```suggestion
        stringBuilder.append(drawUserNames(result.getUserNames()));
        stringBuilder.append(drawBlogDto(result.getBlockDto()));
```

Reply:
객체 지향적으로 코드를 짜다 보니까, getter로 대상을 꺼내서 넘기기 보다는 객체 자체를 넘기는게 습관화 되어있네요, 이쪽은 view를 그리는 쪽이라서 굳이 그렇게 할 필요가 없는거 같습니다. 변경해볼게요